harness/features/allFeatures.local.test.ts

```diff
@@ -34,10 +34,8 @@ describe('allFeatures Tests - Local', () => {
             beforeEach(async () => {
                 testClient = new LocalTestClient(sdkName)
                 scope
-                    .get(
-                        `/client/${testClient.clientId}/config/v1/server/${testClient.sdkKey}.json`,
-                    )
-                    .reply(200, config)
+                    .get(testClient.getValidConfigPath())
+                    .reply(200, testClient.getValidConfig())
```

Comment:
you could also just make this whole scope thing a single function on the client like `respondWithConfig(scope)`

Reply:
I was debating that - how would i pass through the scope/whatnot?

Reply:
just give it as an argument

Reply:
isn't js pass by value not by ref?

Reply:
or there may be a way to retrieve it from somewhere automatically since the scope is set up automatically by the test environment

Reply:
> isn't js pass by value not by ref?
nope

Reply:
yes - but we also override the flow of responding successfully in a few cases - and wanted to keep that functionality

Reply:
yeah  you can still construct the responder manually this would just be a slight convenience for the most common case

Reply:
yeah - gonna do a client function to make it cleaner